configure.ac

```diff
@@ -62,9 +62,10 @@ AC_ARG_WITH(pkg-name,
 
 dnl 1.11:           minimum automake version required
 dnl foreign:        don't require GNU-standard top-level files
+dnl tar-pax:        use POSIX variant of generated tar rather than v7
 dnl silent-rules:   allow "--enable-silent-rules" (no-op in 1.13+)
 dnl subdir-objects: keep .o's with their .c's (no-op in 2.0+)
-AM_INIT_AUTOMAKE([1.11 foreign silent-rules subdir-objects])
+AM_INIT_AUTOMAKE([1.11 foreign tar-pax silent-rules subdir-objects])
```

Comment:
"As of 2018, this format is supported by the native tar command only on GNU, FreeBSD, OpenBSD system; it is not supported by the native tar command on NetBSD, AIX, HP-UX, Solaris." I think tar-ustar would be a better choice for the near future.